validator-api/src/main.rs

```diff
@@ -107,11 +124,26 @@ fn parse_args<'a>() -> ArgMatches<'a> {
             Arg::with_name(WRITE_CONFIG_ARG)
                 .help("specifies whether a config file based on provided arguments should be saved to a file")
                 .long(WRITE_CONFIG_ARG)
+                .short("w")
+        )
+        .arg(
+            Arg::with_name(KEYPAIR_ARG)
+                .help("Path to the secret key file")
+                .takes_value(true)
+                .long(KEYPAIR_ARG)
+        )
+        .arg(
+            Arg::with_name(FIRST_REWARDING_EPOCH_ARG)
+                .help("Datetime specifying beginning of the first rewarding epoch of this length. It must be a valid rfc3339 datetime.")
+                .takes_value(true)
+                .long(FIRST_REWARDING_EPOCH_ARG)
```

Comment:
We could also require that rewarding is enabled:
```suggestion
                .takes_value(true)
                .requires(REWARDING_ENABLED)
                .long(FIRST_REWARDING_EPOCH_ARG)
```

Reply:
Oh. I didn't know we could do that. Nice!